util/data_set_helpers.py

```diff
@@ -0,0 +1,61 @@
+
+import threading
+import numpy as np
+import tensorflow as tf
+
+from util.text import ctc_label_dense_to_sparse
+
+class DataSets(object):
+    def __init__(self, train, dev, test):
```

Comment:
I would not be against a little bit of documentation on this part, it seems not very obvious at a first glance what we are supposed to put into `train`, `dev` and `test`